app/docker-entry.sh

```diff
@@ -21,21 +21,15 @@ if [ "$1" = 'platform' ]; then
     done
 
     echo "Using config file" $MM_CONFIG
-
-    echo -ne "Configure database connection..."
     if [ ! -f $MM_CONFIG ]
     then
-        cp /config.template.json $MM_CONFIG
-        sed -Ei "s/DB_HOST/$DB_HOST/" $MM_CONFIG
-        sed -Ei "s/DB_PORT/$DB_PORT_5432_TCP_PORT/" $MM_CONFIG
-        sed -Ei "s/MM_USERNAME/$MM_USERNAME/" $MM_CONFIG
-        sed -Ei "s/MM_PASSWORD/$MM_PASSWORD/" $MM_CONFIG
-        sed -Ei "s/MM_DBNAME/$MM_DBNAME/" $MM_CONFIG
-        echo OK
-    else
-        echo SKIP
+        cp /config.json.save $MM_CONFIG
     fi
 
+    echo -ne "Configure database connection..."
+    export MM_SQLSETTINGS_DATASOURCE="postgres://$MM_USERNAME:$MM_PASSWORD@$DB_HOST:$DB_PORT/$MM_DBNAME?sslmode=disable&connect_timeout=10"
```

Comment:
I guess it should be since all vars can be override : https://docs.mattermost.com/administration/config-settings.html